src/ui/molecules/stripe-express-checkout-element.svelte

```diff
@@ -91,6 +87,8 @@
     expressCheckoutElement?.destroy();
     expressCheckoutElement = null;
   });
+
+  // TODO: find a way to update the wallet config when expressCheckoutOptions get updated.
```

Comment:
This is mandatory for the tax-exclusive scenario. We will need an effect like the one we have for `elementsConfiguration`:

```
$effect(() => {
    if (expressCheckoutConfiguration && elements) {
      StripeService.updateExpressCheckoutConfiguration(
        elements,
        expressCheckoutConfiguration,
      );
    }
  });
```